src/app/core/services/notifications/fcm-rest-notification.service.ts

```diff
@@ -0,0 +1,186 @@
+import { Injectable } from '@angular/core'
+import { Firebase } from '@ionic-native/firebase/ngx'
+import { Platform } from 'ionic-angular'
+import * as Swagger from 'swagger-client'
+
+import {
+  DefaultMaxUpstreamResends,
+  DefaultPackageName,
+  DefaultSourcePrefix
+} from '../../../../assets/data/defaultConfig'
+import { SingleNotification } from '../../../shared/models/notification-handler'
+import { TaskType } from '../../../shared/utilities/task-type'
+import { getSeconds } from '../../../shared/utilities/time'
+import { RemoteConfigService } from '../config/remote-config.service'
+import { SubjectConfigService } from '../config/subject-config.service'
+import { LocalizationService } from '../misc/localization.service'
+import { LogService } from '../misc/log.service'
+import { ScheduleService } from '../schedule/schedule.service'
+import { StorageService } from '../storage/storage.service'
+import { FcmNotificationService } from './fcm-notification.service'
+import { NotificationGeneratorService } from './notification-generator.service'
+
+@Injectable()
+export class FcmRestNotificationService extends FcmNotificationService {
+  private APP_SERVER_URL = 'http://localhost:8080'
```

Comment:
should this be taken from remote config?

Reply:
Ok but by default would it be on Rosalind? `https://radar-cns-platform.rosalind.kcl.ac.uk/appserver`?

Reply:
I am not sure right now. Probably will be a different domain name. But let's revisit once we have a deployment. But we can change that via remote config if required.

Reply:
Ok I've added it to remote config.